scripts/rabbitmq-service.bat

```diff
@@ -188,6 +188,16 @@ set ERLANG_SERVICE_ARGUMENTS= ^
 set ERLANG_SERVICE_ARGUMENTS=!ERLANG_SERVICE_ARGUMENTS:\=\\!
 set ERLANG_SERVICE_ARGUMENTS=!ERLANG_SERVICE_ARGUMENTS:"=\"!
 
+set ENV_OK=true
+CALL :check_not_empty "RABBITMQ_BOOT_MODULE" "!RABBITMQ_BOOT_MODULE!" 
+CALL :check_not_empty "RABBITMQ_NAME_TYPE" "!RABBITMQ_NAME_TYPE!"
+CALL :check_not_empty "RABBITMQ_NODENAME" "!RABBITMQ_NODENAME!"

```

Comment:
I removed the double quotes also here, for the same reason I did [here] (https://github.com/rabbitmq/rabbitmq-server/commit/053e798b312a01ada29c17cc101fa140f3ae9753)